system/help/help.js

```diff
@@ -0,0 +1,188 @@
+	var padding = 10;
+	var x = anchor.clientX; // Get the horizontal coordinate
+	var y = anchor.clientY; // Get the vertical coordinate
+	// determine if window is scrolled
+	if (window.pageXOffset !== undefined) {
+		// All browsers, except IE9 and earlier
+		x += window.pageXOffset;
+		y += window.pageYOffset;
+	} else { // IE9 and earlier
+		x += document.documentElement.scrollLeft;
+		y += document.documentElement.scrollTop;
+	}
+	// get viewport
+	var sx = document.documentElement.clientWidth;
+	var sy = document.documentElement.clientHeight;
+	var windowCenter = sx / 2;
+	var box = measureText(text, fontSize, fontStyle, padding);
+	$textBox.css('fontSize', fontSize + 'px');
+	$textBox.css('fontStyle', fontStyle);
+	$textBox.css('padding', padding + 'px');
+	$textBox.css('width', box.width + 'px');
+	$textBox.css('height', box.height + 'px');
+	$textBox.html(text);
+	if ((y < box.height) && (x < box.width)) {
+		$textBox.css('left', x + box.width / 3 + 'px');
+		$textBox.css('top', y + 15 + 'px');
+		$textBox.addClass('notelo');
+	} else if ((y < box.height) && ((box.width < x) && (x < sx - box.width))) {
+		$textBox.css('left', x + 'px');
+		$textBox.css('top', y + 15 + 'px');
+		$textBox.addClass('notelo up');
+	} else if ((y < box.height) && (x > sx - box.width)) {
+		$textBox.css('left', x - box.width + 'px');
+		$textBox.css('top', y + 15 + 'px');
+		$textBox.addClass('notelo');
+	} else if (((y >= box.height) && (y < sy - box.height)) && (x < windowCenter)) {
+		$textBox.css('left', x - 40 + box.width / 2 + 'px');
+		$textBox.css('top', y - 10 - box.height / 2 + 'px');
+		$textBox.addClass('notelo left');
+	} else if (((y >= box.height) && (y < sy - box.height)) && (x > windowCenter)) {
+		$textBox.css('left', x - box.width + 'px');
+		$textBox.css('top', y - 10 - box.height / 2 + 'px');
+		$textBox.addClass('notelo right');
+	} else if ((y >= box.height) && (x < box.width)) {
+		$textBox.css('left', x + box.width / 3 + 'px');
+		$textBox.css('top', y - 10 - box.height + 'px');
+		$textBox.addClass('notelo');
+	} else if ((y >= box.height) && ((box.width < x) && (x < sx - box.width))) {
+		$textBox.css('left', x - box.width / 3 + 'px');
+		$textBox.css('top', y - box.height - 40 + 'px');
+		$textBox.addClass('notelo down');
+	} else if ((y >= box.height) && (x > sx - box.width)) {
+		$textBox.css('left', x - box.width + 'px');
+		$textBox.css('top', y - box.height - 15 + 'px');
+		$textBox.addClass('notelo');
+	} else alert('gebied niet te bepalen');
+}
+
+/**
+ * Handy JavaScript to measure the size taken to render the supplied text;
+ * you can supply additional style information too if you have it.
+ *
+ * @param text string
+ * @param fontSize number
+ * @param fontStyle string, the font style
+ * @param padding number
+ * @param border, boolean, indicating whether or not the include a 1px border
+ * @returns {{width: number, height: number}}
+ */
+function measureText(text, fontSize, fontStyle, padding, border = false) {
```

Comment:
Why measure the size of the text in a separate div, then thrash it and then put the same text in the actual infobox div? The code creating this div here needs to be identical to the final div to make the prediction accurate, so might as well measure in the actual div?